dcm2niix_gear/dcm2niix/arrange.py

```diff
@@ -118,6 +118,126 @@ def exit_if_archive_empty(archive_obj):
         os.sys.exit(1)
 
 
+def tally_files(dir_loc, print_out=False):
+    """Function to profile directory. Note current exclusion of {'\.DS_Store.*', '^\._.*'}
+    from file_set and file_tree. # TODO move this exclusion?
+
+    Args:
+        dir_loc (str): path to directory to profile
+        print_out (bool): log output (Default: False)
+    Returns:
+        file_set (set): set of file leaves that pass the regex criteria
+        file_tree (list): list showing full filepaths of all files
+        file_name_path_dict (dict): dict mapping from filenames that pass
+         the regex criteria to full filepath
+    """
+
+    file_set = set()
+    file_name_path_dict = {}
+    file_tree = []
+    gen = os.walk(dir_loc, topdown=True)
+    next_gen = next(gen)
+    # want to exclude files matching certain patterns str_to_exclude
+    strs_to_exclude = ["\.DS_Store.*", "^\._.*"]
```

Comment:
I still think the `^\..*` makes more sense to me to exclude all hidden files/dir.

Reply:
Yes, sorry, you're absolutely right, changed to
strs_to_exclude = ["^\..*"]